app/metrics.yaml

```diff
@@ -0,0 +1,80 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+no_lint:
+  - CATEGORY_GENERIC
+
+metrics:
+  default_browser:
+    type: boolean
+    lifetime: application
+    description: |
+      Is Focus the default browser?
+    send_in_pings:
+      - metrics
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+      - TBD
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - TBD
+    expires: "2021-08-01"
+  search_engine:
+    type: string
+    lifetime: application
+    description: |
+      A string containing the default search engine.
+    send_in_pings:
+      - metrics
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - TBD
+    expires: never
+  locale_override:
+    type: string
+    lifetime: application
+    description: |
+      Locale override
+    send_in_pings:
+      - metrics
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - TBD
+    expires: never
+
+activation:
+  activation_id:
+    type: uuid
+    lifetime: user
+    description: |
+      An alternate identifier, not correlated with the client_id, generated once
+      and only sent with the activation ping.
+    send_in_pings:
+      - activation
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - highly_sensitive
+    notification_emails:
+      - TBD
+    expires: "2021-08-01"
+    no_lint:
+      - USER_LIFETIME_EXPIRATION
```

Comment:
Why do you need this lint suppression? Does removing the expiration work?

Reply:
Sorry, this was just some bare metrics copied over from Fenix without thought for the sake of testing the code. :)